### ISSUE TITLE
Docs version has not been updated

### ISSUE THREAD

**Issue by phlax** [envoyproxy/envoy-website#260]
not sure if this is a failed automation, or something in need of it

**Comment by phlax** [comment]
cc @moderation

**Comment by moderation** [comment]
I'm thought updating the website was part of a release checklist but not automated - here are my prior PRs - https://github.com/envoyproxy/envoy-website/pulls?q=is%3Apr+is%3Aclosed+author%3Amoderation

**Comment by phlax** [comment]
yep - sure - but i shifted the build to bazel with the intention of automating anything possible - seems like i missed this - shouldnt be too difficult

we can update in the meantime tho

**Comment by phlax** [comment]
i think this was resolved a long time back
